example/index.js

```diff
@@ -70,6 +70,22 @@ db.installUsersBehavior().then(function () {
     }
   })
 
+  server.route({
+    method: 'GET',
+    path: '/default/css/default.css',
+    handler: function (request, reply) {
+      reply.file(path.join(__dirname, '..', 'public', 'client.css'))
+    }
+  })
+
+  server.route({
+    method: 'GET',
+    path: '/default/js/default.js',
+    handler: function (request, reply) {
+      reply.file(path.join(__dirname, '..', 'public', 'client.js'))
+    }
+  })
```

Comment:
Instead of defining separate paths for each static asset, let’s use [inert](https://www.npmjs.com/package/inert) to server all assets from the `public/` folder at the `/` root path.